content/docs/dashboard/features.md

```diff
@@ -16,32 +16,68 @@ The user interface is still under development and therefore does not offer so ma
 However, we are working on it to support more and more features. 
 If you find ideas for new features or bugs post an issue in the [GitHub repository](https://github.com/oakestra/dashboard).
 
+
+## Creating an application
+
+In Oakestra there is the principle of applications and services, within one application there can be several services and one user can create several applications.
+
+In order to create a service, an application must first be created.  This is very simple, you only need to specify the name, the namespace and an optional description.
+
+![](/add-app.gif)
+
 ## Creating a service
 
-Creating a service is the main task of the dashboard, as described in other parts of the documentation, 
-an SLA definition must be created for it. This can be easily done using the various input fields 
-in the dashboard. 
+Creating a service is the main task of the dashboard, as described in other parts of the documentation,
```

Comment:
Can you add reference to https://www.oakestra.io/docs/getstarted/get-started-app/ highlighting that it is possible to deploy application from the API browser as well but the dashboard makes everything a "one stop shop"?